scripts/README.md

```diff
@@ -2,25 +2,81 @@
 
 This directory contains utility scripts for the SIGNAL web app.
 
-## Admin Setup
-
-Requires Node 16+. To check, run the following command.
-```bash
-$ node -v
-```
+## Table of Contents
+1. [Tools Setup](#tools-setup)
+2. [Setup Signal](#setup-signal)
+3. [Set up for running the admin scripts](#set-up-for-running-the-admin-scripts)
+4. [Run the upload script](#upload-data)
 
+## Tools Setup
+1. [Download Visual Studio Code](https://code.visualstudio.com/) or use whichever code editor you'd prefer.
+2. [Download Node version 16+](https://nodejs.org/en/download/)
+    - To check if you have node installed, open a terminal window and run the following command.
+    ```bash
+    node -v
+    ```
+3. For MacOS, install xcode command line tools
+   - Check if you have it already installed. In your terminal run the following command.
+     ```
+     xcode-select -p
+     ```
+     If you see an error, the package has not been installed otherwise you're good to go.
+   - To install xcode command line tools run the following command.
+     ```
+     xcode-select --install
+     ```
+4. Install Firebase Tools
+   - Run the following command in terminal
+     ```
+     sudo npm install -g firebase-tools
+     ```
+     
+## Setup Signal
+1. In terminal, navigate to the folder you want to put the `signal-app` project into.
+    - [Terminal for beginners tutorial!](https://medium.com/@grace.m.nolan/terminal-for-beginners-e492ba10902a)
+        - Only really need `ls`, `cd`, `pwd`
+        - Use `ls` command to see the folder you're in, and then `cd` to navigate.
+            - `ls` lists files in current directory
+            - `cd` change directory
+        - Mine is at `/Users/ellen/Documents/Projects` so I could do `cd /Users/ellen/Documents/Projects`. 
+          Which is the absolute path to my project directory.
+2. Go to the [signal-app repository](https://github.com/pph-collective/signal-app)
+3. Clone the repository. 
+    - On the page you'll see a green "Code" button, click it and copy the https `https://github.com/pph-collective/signal-app.git`
+    - In the terminal, run `git clone` with the https you just copied. The full command is as follows:
+      ```
+      git clone https://github.com/pph-collective/signal-app.git
+      ```
+      This will create the directory `signal-app`.
+4. Open VSCode and open the folder `signal-app`
```

Comment:
```suggestion
4. Use `cd signal-app` to switch the the repo directory and then `code .` to open VSCode in the folder `signal-app`
```

Reply:
i think it would be easier for our users to navigate the through vscode instead of trying to find the signal-app directory through the terminal